packages/webpack/internal/worker/worker_adapter.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * @fileoverview wrapper program around the Webpack CLI.
+ *
+ * It intercepts the Bazel Persistent Worker protocol, using it to
+ * remote-control webpack cli. It tells the compiler process to
+ * consolidate file changes only when it receives a request from the worker
+ * protocol.
+ *
+ * See https://medium.com/@mmorearty/how-to-create-a-persistent-worker-for-bazel-7738bba2cabb
+ * for more background on the worker protocol.
+ */
+import * as worker from "@bazel/worker";
+import * as cp from 'child_process';
+import * as fs from 'fs';
+
+const webpackCli = require("webpack-cli");
+
+type IPCMessage = {
+  type: "built" | "error" | "ready"
+  [key: string]: unknown
```

Comment:
I don't think you need this particular line

Reply:
Nice catch!